src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/RefreshVendorRegistrationCaseStatus.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities
+{
+    public class RefreshVendorRegistrationCaseStatus
+    {
+        private readonly IVendorRegistrationFormService _vendorRegistrationFormService;
```

Comment:
"Form" service?  Isn't it just the VendorRegistrationService  ?

Reply:
VRF

Reply:
VRF Service